Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -410,3 +410,59 @@ label monika_monikamoddev:
     if not wrs_success:
         $ mas_unlockFailedWRS('monika_monikamoddev')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_4chan",
+            category=['4chan'],
+            rules={"notif-group": "Window Reactions", "skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_4chan:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "So this is the place where this all started, huh?\nIt's...really quite something.",
+            "Just don't end up arguing with other Anons all day long, [player]."
+            "I heard you guys have threads discussing our Literature Club in here.\nTell them I said hi~"
+            "I'll be watching the boards you're browsing... in case you get any ideas, ahaha!"
+        ],
+        'Window Reactions'
+    )
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('mas_wrs_4chan')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_pixiv",
+            category=['pixiv'],
+            rules={"notif-group": "Window Reactions", "skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_pixiv:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "I'm curious as to what people have come up with... Mind looking for some art of me?\nJust keep it wholesome, though~",
+            "This is a pretty interesting place... so much skilled people posting their work.\nAre you one of them, [player]?"
```

Comment:
This one here, problem with this is that Monika asks you if you've drawn art of her, so if you answered no, it would be odd for her to essentially ask it again. I've got an idea on how to remedy this though. Do you mind if I push to your branch?

Reply:
Go on.